Source/BoundaryConditions/PML.H

```diff
@@ -43,13 +43,15 @@ struct SigmaBox
 {
     SigmaBox (const amrex::Box& box, const amrex::BoxArray& grids,
               const amrex::Real* dx, const amrex::IntVect& ncell, const amrex::IntVect& delta,
-              const amrex::Box& regdomain);
+              const amrex::Box& regdomain, amrex::Real v_sigma);
```

Comment:
Since this new function argument is read-only, I think it would be appropriate to add the `const` qualifier (same comment holds for similar instances below):
```suggestion
              const amrex::Box& regdomain, const amrex::Real v_sigma);
```